### PR TITLE
KOGITO-5381 Merge create branch and prepare release

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-5381

Extra job is quite useless and confusing so merging it into the main one.

### CODE REVIEW

File: .jenkins/dsl/jobs.groovy [hunk 1]
```diff
@@ -193,27 +192,11 @@ void setupReleaseJob(String jobFolder) {
...
-        }
-    }
-}
-
 void setupPrepareReleaseJob(String jobFolder) {
     KogitoJobTemplate.createPipelineJob(this, getJobParams('prepare-release-branch', jobFolder, 'Jenkinsfile.release.prepare', 'Prepare env for a release')).with {
         parameters {
             stringParam('KOGITO_VERSION', '', 'Project version to release as Major.minor.micro')
             stringParam('OPTAPLANNER_VERSION', '', 'Project version of OptaPlanner and its examples to release as Major.minor.micro')
-            stringParam('OPTAPLANNER_RELEASE_BRANCH', '', 'Use to override the release branch name deduced from the OPTAPLANNER_VERSION')
```
Comment (radtriste) [inline .jenkins/dsl/jobs.groovy:216]: Removed as we never use it ...

**Comment by radtriste** [comment]
tested all ok !

**Comment by radtriste** [comment]
@rsynek @mbiarnes @msmagnanijr Would be great to merge this one before the 1.9.x/8.9.x branching next week :)

**Review by rsynek — APPROVED** [review]

**Review by mbiarnes — APPROVED** [review]
+1

**Pull request merged**